frontend/src/views/WheresMyGene/components/HeatMap/components/Chart/index.tsx

```diff
@@ -306,9 +306,13 @@ export default memo(function Chart({
       ],
     };
 
+    const cellTypeName = cellType.cellTypeName.startsWith("UBERON")
```

Comment:
to avoid confusion, should we rename `cellType` to `row` or something better, and add notes that row can be either a tissue or a cellType, so it's clearer why `cellType.cellTypeName` can be a tissueId?

Thank you!